src/plugins/content/accessibility/jumplinks/shortcuts.tpl

```diff
@@ -0,0 +1,85 @@
+<div class="shortcuts-list-wrapper hidden">
+    <div class="shortcuts-list">
+        <h1 id="keyboard-navigation">Keyboard Navigation</h1>
```

Comment:
Should the string in the template be translated? Also using h1 should be discussed with UX